testing/testcafe/helpers/screenshot-comparer.ts

```diff
@@ -270,16 +267,22 @@ export async function compareScreenshot(
   element: SelectorType = null,
   comparisonOptions?: Partial<ComparerOptions>,
 ): Promise<boolean> {
+  const { path: rootPath, ...configOptions } = (t as unknown as { testRun }).testRun.opts['screenshots-comparer'] as ComparerOptions;
+  const testRoot = rootPath ?? './testing';
```

Comment:
A developer can pass the `path` through the `comparisonOptions` argument. 
I'm not also sure if we need the coalescing operator here: `    ...comparisonOptions ?? {}`. Even if comparisonOptions is `undefined`, everything will be ok.
```suggestion
  const configOptions = (t as unknown as { testRun }).testRun.opts['screenshots-comparer'] as ComparerOptions;
  const options = {
    ...screenshotComparerDefault,
    ...configOptions,
    ...comparisonOptions,
  } as ComparerOptions;
  const testRoot = options.path;
```